docs/dev/ci/github_actions/docker_images.md

```diff
@@ -44,39 +44,133 @@ Here are some pros and cons of having your own container registry:
     * The registry does not automatically mirror images available on Docker Hub
     * The necessary images must be manually added to the registry
 
-The available Docker images use the following pattern for their names: `openvinogithubactions.azurecr.io/dockerhub/<image-name>:<image-tag>`.
+The available Docker images use the following pattern for their names: `openvinogithubactions.azurecr.io/<image-name>:<image-tag>`.
 
-Most of the images on the OpenVINO ACR are mirrors of the images with the same names on Docker Hub.
+Most of the images on the OpenVINO ACR are either mirrors of the images from Docker Hub or based on them.
 
 Examples:
-* `openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04` corresponds to `ubuntu:20.04` from Docker Hub
-* `openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04` corresponds to `ubuntu:22.04` from Docker Hub
-* `openvinogithubactions.azurecr.io/dockerhub/nvidia/cuda:11.8.0-runtime-ubuntu20.04` corresponds to `nvidia/cuda:11.8.0-runtime-ubuntu20.04` from Docker Hub
+* `openvinogithubactions.azurecr.io/library/ubuntu:22.04` corresponds to `ubuntu:22.04` from Docker Hub
```

Comment:
Does it not contradict/not in line with the pattern in https://github.com/openvinotoolkit/openvino/pull/26597/files#diff-5fa037687dbf0e9e3eea790196bb01869668492532141efcf85a78eb307a34c8R47? I think the exceptions could be noted as a note under the pattern.

Reply:
It's a bit more complicated actually - everything that is available in dockerhub under `docker.io/library/<image>:<tag>`, like standard ubuntu images, will be available in our registry as `openvinogithubactions.azurecr.io/<image>:<tag>`; and for images like cuda which are not under library prefix `docker.io/<image>:<tag>` -> `openvinogithubactions.azurecr.io/<image>:<tag>`, so in fact we just replace docker.io with openvinogithubactions.azurecr.io. The difficulty here is to find out which images do have /library prefix and which are not, since dockerhub allows to skip library prefix alltogether, while our mirroring rules can't manage that

Reply:
Added the "library" variant to the patterns without specifying details